src/io/file_descriptor.cr

```diff
@@ -18,6 +18,27 @@ class IO::FileDescriptor < IO
     end
   end
 
+  def self.from_stdio(fd)
```

Comment:
This method should probably be `:nodoc:`, it's only internal implementation and probably not useful in user code.

Reply:
:+1: